pkg/volume/volume.go

```diff
@@ -63,6 +63,10 @@ type Attributes struct {
 	ReadOnly        bool
 	Managed         bool
 	SupportsSELinux bool
+	// This will be set if a volume has a GID annotation which is
+	// useful if the volume does not support ownership management
+	// and can only be read as that GID
+	VolumeGID *int64
```

Comment:
> per our IRC chat, i think there is agreement that using volumeOptions is better than Attributes.

@markturansky it looks like even that is unnecessary since we have access to the pod right in NewBuilder in persistent_claim.go